test/studies/shootout/nbody/bradc/nbody-blc.chpl

```diff
@@ -1,15 +1,15 @@
 /* The Computer Language Benchmarks Game
    https://salsa.debian.org/benchmarksgame-team/benchmarksgame/
 
-   contributed by Albert Sidelnik and Brad Chamberlain
-   derived from the Java version by Mark C. Lewis and Chad Whipkey
+   contributed by Brad Chamberlain
+   derived from the Chapel version by Albert Sidelnik and myself
 */
 
 
 config const n = 10000;       // The number of timesteps to simulate
 
-const pi = 3.141592653589793,
-      solarMass = 4 * pi**2,
+param pi = 3.141592653589793,
```

Comment:
Is there any reason we don't use our `Math.pi`?

Reply:
Good point.  Not that I'm aware of, and it looks like it works with the other pi.  I'm going to add a `use Math;` to make this future-proof in case `pi` doesn't make it into the "`AutoMath`" module.